pennylane/qchem/basis_set.py

```diff
@@ -66,16 +66,18 @@ def __init__(self, l, alpha, coeff, r):
         self.params = [self.alpha, self.coeff, self.r]
 
 
-def atom_basis_data(name, atom):
+def atom_basis_data(name, atom, load_data=False):
```

Comment:
Soran suggested informally while we discussed that perhaps we could remove `load_data` and simply use basis-set-exchange by default if the string specifying the basis is not natively supported in pennylane, issuing an error message if the user does not have basis set exchange loaded. It's a good idea worth considering, but for now I am inclined to say that having `load_data` is good